packages/components/input/src/Input.doc.mdx

```diff
@@ -104,6 +98,12 @@ Please note that the `TrailingIcon` will be automatically replaced by the state
 
 <Canvas of={stories.Icons} />
 
+## Disabled
+
+Use `disabled` prop to indicate the input is disabled (set the prop on `InputGroup` instead if you are wrapping your `Input` with it)
```

Comment:
Suggestion:
To indicate that an input is disabled, use the `disabled` prop. If you're wrapping your `Input` with an `InputGroup`, set the prop on the `InputGroup` instead.